### DISCUSSION TITLE
Statistics dashboard ?

### DISCUSSION THREAD

**Discussion by Kikimahe** [seerge/g-helper#1958] [Q&A] [answered]
Hi Seerge, 

I am naturally curious...  :)

A few days ago you said that there were 70,000 Ghelper users. 🥇 
I think this number increased with AC's AacAmbientLighting.exe error...

Does a statistics dashboard exist on Ghelper users ?
(Total users, versions used, , laptop model, CPU, GPU, etc.)

Usually I don't like telemetry and you would certainly be the target of further criticism.
But maybe some of his anonymous information could be displayed on an online dashboard, by Ghelper?

Sorry if the questions are stupid, but it tickles my mind. :)

Have a nice day.

**Comment by seerge** [comment, marked as answer]
@Kikimahe G-Helper doesn't have any tracking inside, obviously :) 

I was doing a poll about models [some time ago](https://github.com/seerge/g-helper/discussions/707) and I believe it's still representative 

Next to that I just track downloads stats for "final" releases 
https://hanadigital.github.io/grev/?user=seerge&repo=g-helper

And if I don't do updates too frequently, it usually collects 50-60K downloads (like 0.141, 0.134, etc). Considering that lots of people simply don't update app at all, I estimate monthly actives to something like 70K indeed.

Github itself has very poor stats, only visits to web-page itself (that don't include downloads)
![image](https://github.com/seerge/g-helper/assets/5920850/c3ae8e3b-8f71-4c60-93dd-bb5a10e69714)

**Comment by Kikimahe** [comment]
Thank you for all these elements.
(I knew #707)

It's very interesting. 👍
